### DISCUSSION TITLE
View available colormaps for previous versions

### DISCUSSION THREAD

**Discussion by nkleinbaer** [cogeotiff/rio-tiler#544] [Q&A] [answered]
Hi, all, I'm wondering if there is a way to view what color maps are available for previous versions? 

I see [this list of default color maps](https://cogeotiff.github.io/rio-tiler/colormap/#default-rio-tilers-colormaps) from in the docs, but I am working on application that uses a older version (`2.0.1`), I'm wondering if there is a way to have the library list out what CMs are registered. 

Thanks!

**Comment by vincentsarago** [comment]
I don't think they changed since 2.0

**Reply by nkleinbaer** [reply]
Thanks for prompt reply! That's odd though, we are trying to use `RdYlGr` which should be a standard one from matplotlib, and it's raising `InvalidColorMapName` ... looking a bit closer though it's looking like the cmap name got cast to lowercase at somepoint, that might be a mistake on our side.

**Reply by nkleinbaer** [reply]
🤦 ... it should have been `RdYlGn`, not `RdYlGr` ... sorry to have wasted your time, and thanks again!

**Comment by nkleinbaer** [comment, marked as answer]
Incase anyone stumbles on this in the future, I was able to check the registered color maps in my current version by doing the following:

```python
from rio_tiler.colormap import cmap

valid_cmaps = [key for key in cmap.data.keys()]
valid_cmaps.sort()
print(valid_cmaps)
```
